tools/pot/tests/test_graph.py

```diff
@@ -25,7 +25,8 @@
     ('densenet121_example', 'pytorch', 'ANY'),
     ('multiple_out_ports_net', 'tf', 'ANY'),
     ('lstm_example', 'pytorch', 'GNA'),
-    ('multiple_outputs_net_example', 'dldt', 'GNA')
+    ('multiple_outputs_net_example', 'dldt', 'GNA'),
+    ('tensor_iterator_example', 'dldt', 'ANY')
```

Comment:
Can we have a framework version of the model (.onnx, .pb, etc.)? The reason is that from time to time IR is changed so we need to recreate references. But it will be problematic if we don't have an original model for that. Just placing an original model and calling MO every time looks better.